test/e2e/utils/recordset-utils.ts

```diff
@@ -87,12 +90,40 @@ export async function testSelectFacetOption(page: Page, facet: Locator, optionId
  * @param optionIdx facet option index to check is unchecked
  * @param pageSize the recordset page size for comparing with after clear
  */
-export async function testClearAllFilters(page: Page, facet: Locator, optionIdx: number, pageSize: number) {
+export async function testClearAllFilters(page: Page, pageSize: number, facet?: Locator, optionIdx?: number) {
   const clearAll = RecordsetLocators.getClearAllFilters(page);
   await clearAll.click();
   await expect.soft(clearAll).not.toBeVisible();
-  await expect.soft(RecordsetLocators.getFacetOption(facet, optionIdx)).not.toBeChecked();
   await expect.soft(RecordsetLocators.getRows(page)).toHaveCount(pageSize);
+
+  if (facet && optionIdx) await expect.soft(RecordsetLocators.getFacetOption(facet, optionIdx)).not.toBeChecked();
+}
+
+export async function testShowMoreClick(page: Page, facet: Locator, modal: Locator, numRows: number, numCheckedRows: number) {
```

Comment:
Can you please add jsdoc comment to these functions that you've added?

For example for this function you should mention that it will open the show more modal and leave it open.